src/devices/Ssd1351/Ssd1351Commands.cs

```diff
@@ -0,0 +1,541 @@
+        {
+            SendCommand(Ssd1351Command.SetDisplayOff);
+        }
+
+        /// <summary>
+        /// This command enhances display performance. 
+        /// </summary>
+        /// <param name="enhanceDisplay">When set to true turns on enhanced display mode.</param>
+        public void SetDisplayEnhancement(bool enhanceDisplay = false)
+        {
+            SendCommand(Ssd1351Command.SetDisplayEnhancement, (byte)(enhanceDisplay ? 0xA4 : 0x00), 0x00, 0x00);
+        }
+
+        /// <summary>
+        /// This double byte command is used to set the phase 3 second pre-charge period.  The period of phase 3 is ranged from 1 to 15 DCLK's.
+        /// </summary>
+        /// <param name="phase3Period">Phase 3 period with a range of 1-15.</param>
+        public void Set3rdPreChargePeriod(byte phase3Period = 0x08)
+        {
+            if (!Ssd1351.InRange(phase3Period, 0x01, 0x0F))
+            {
+                throw new ArgumentException("The phase 3 period is invalid.", nameof(phase3Period));
+            }
+
+            SendCommand(Ssd1351Command.SetPrecharge2, phase3Period);
+        }
+
+        /// <summary>
+        /// This double byte command is used to set the pre-charge voltage level. The precharge
+        /// voltage level ranges from 0.20 x Vcc -> 0.60 x Vcc.
+        /// </summary>
+        /// <param name="prechargeLevel">Pre-charge voltage level with a range of 0-31 that represents 0.20 x Vcc -> 0.60 x Vcc.</param>
+        public void SetPreChargeVoltageLevel(byte prechargeLevel = 0x17)
+        {
+            if (!Ssd1351.InRange(prechargeLevel, 0x00, 0x1F))
+            {
+                throw new ArgumentException("The pre-charge voltage level is invalid.", nameof(prechargeLevel));
+            }
+
+            SendCommand(Ssd1351Command.SetPrechargeVoltageLevel, prechargeLevel);
+        }
+
+        /// <summary>
+        /// This triple byte command specifies column start address and end address of the display data RAM.
+        /// This command also sets the column address pointer to column start address. This pointer is used
+        /// to define the current read/write column address in graphic display data RAM. If horizontal address
+        /// increment mode is enabled by command 20h, after finishing read/write one column data, it is
+        /// incremented automatically to the next column address. Whenever the column address pointer finishes
+        /// accessing the end column address, it is reset back to start column address and the row address
+        /// is incremented to the next row.  This command is only for horizontal or vertical addressing modes.
+        /// </summary>
+        /// <param name="startColumn">Column start address with a range of 0-127.</param>
+        /// <param name="endColumn">Column end address with a range of 0-127.</param>
+        public void SetColumnAddress(byte startColumn = 0x00, byte endColumn = 0x7F)
+        {
+            if (startColumn > 0x7F)
```

Comment:
You are only checking for upper bound not for lower bounds. Same goes for endColumn.

Reply:
if the lower boundary is 0 then anything below will underflow and will satisfy the check